### PR TITLE
Bind paramaters to query with pystache instead of str.format

### PR DESCRIPTION
**Pull request by ariarijp**
Hi, @toritori0318 

I want to solve an issue related to non-parameter curly brace.
e.g. REGEXP() on MySQL or REGEXP_MATCH() on BigQuery and so on.

If you have any concerns, please feel free to let me know.
Thanks.

### CODE REVIEW

File: redash_dynamic_query/__init__.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import time
 
+import pystache
```
Comment (ariarijp) [inline redash_dynamic_query/__init__.py:3]: The reason why I choose pystache, It's because Redash uses pystache currently.

Comment (toritori0318) [reply redash_dynamic_query/__init__.py:3]: 👌

**Comment by toritori0318** [comment]
Thanks! 👍

**Comment by toritori0318** [comment]
published. > https://testpypi.python.org/pypi/redash-dynamic-query/1.0.1

**Comment by toritori0318** [comment]
published > https://pypi.python.org/pypi/redash-dynamic-query/1.0.1

**Pull request merged**